components/display/tm1638.rst

```diff
@@ -164,5 +163,7 @@ See Also
 --------
 
 - :doc:`index`
+- :doc:`switches </components/switch/index>`
```

Comment:
The first part gives the text to use for the link.  In this case, you don't need that. So just:
:doc:\`/components/switch/index\`

Reply:
Removed unnecessary word 'switches'